build-faq.md

```diff
@@ -4,4 +4,58 @@ When building on macOS, the build system will make a native Cocoa app by default
 
 # Attempting to build the SDL frontend on macOS fails on linking
 
-SameBoy on macOS expects you to have SDL2 installed via Brew, and not as a framework. Older versions expected it to be installed as a framework, but this is no longer the case.
\ No newline at end of file
+SameBoy on macOS expects you to have SDL2 installed via Brew, and not as a framework. Older versions expected it to be installed as a framework, but this is no longer the case.
+
+# Windows build process
+
+For the various tools and libraries, follow the below guide to ensure easy, proper configuration for the build environment:
+
+#### clang
+
+This may be installed via a Visual Studio installer packages instead of built from source.
+
+#### SDL Port
+
+[libsdl2](https://libsdl.org/download-2.0.php) has two separate files that must be downloaded
+  1. The `-x86` Runtime Binary (e.g., `SDL2-2.0.12-win32-x86.zip` (as of writing))
```

Comment:
Why do you need download both? The DLL file is already included in the development zip.

Reply:
I hadn't realized that the DLL was also within the \x86\ folder. Will resolve this and address the %path% concern in the other thread.